dc.stage.yml

```diff
@@ -24,6 +24,9 @@ services:
       - FPLM_DB=fpmaps_d7_live
       - FPLM_USER=
       - FPLM_PW=
+      - EMAIL_HOST=smtp.sendgrid.net
+      - EMAIL_HOST_USER=countable
+      - EMAIL_HOST_PASSWORD=4UvuL4u9
```

Comment:
@jrcarretas this is a big problem. You've published a secret API key to a public repository.

Reply:
Before you merge this, please delete the value from the history and force push to your branch. You can do this by squashing all your commits including the commit that created this value, and force pushing to this branch.